.github/prompts/code-rabbit-review.md

```
@@ -1,10 +1,17 @@
-CodeRabbit review on PR #$NUMBER (Branch: $BRANCH).
+$PROJECT_CONTEXT
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)